src/state/stake/stake-constants.ts

```diff
@@ -112,10 +111,10 @@ const NULL_POOL: StakingTri = {
   apr: 0,
   nonTriAPRs: [],
   chefVersion: ChefVersions.V1,
-  doubleRewards: false,
+  hasNonTriRewards: false,
   inStaging: false,
   noTriRewards: false,
-  doubleRewardToken: dummyToken,
+  earnedNonTriRewards: [],
```

Comment:
This makes adding pools easier, but if a user has farmed tokens that
1. haven't been claimed
2. rewards are turned off

They'll only know if they look at the farm card